server/server.go

```diff
@@ -642,6 +642,9 @@ func (s *Server) handleFile(w http.ResponseWriter, r *http.Request, v *visitor)
 		return err
 	}
 	defer f.Close()
+	if m.Attachment.Name != "" {
+		w.Header().Set("Content-Disposition", "attachment; filename="+strconv.Quote(m.Attachment.Name))
+	}
```

Comment:
Wait, ... This changes the default browser behavior, right? Will images be downloaded too, or just things that are normally downloaded?

Reply:
Everything will download automatically if you go directly to the URL, including images. When would you go to the URL and not want to download it? Just to see a bigger version of the image?

We could add a condition based on content type

Reply:
Well, the web app embeds images. I am not sure if there are other use cases, but I'd say it's safe to say we'll want to inline display images at least. Hm....

Reply:
I'm 90% sure the embedded/inline images worked just fine in the web app when I tested it

Reply:
Oh interesting. Maybe the browser does the right thing. I'll test.

Reply:
Works indeed.